Source/WebCore/platform/network/soup/ResourceRequestSoup.cpp

```diff
@@ -140,6 +140,23 @@ void ResourceRequest::updateSoupMessageBody(SoupMessage* soupMessage, BlobRegist
 
 }
 
+GRefPtr<GInputStream> ResourceRequest::createInputStream() const
+{
+    auto formData = httpBody();
```

Comment:
Heh, so it is still serialized after all? We were previously serializing it twice prior to https://commits.webkit.org/260252@main? OK.